packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DomainBuilderHelper.ts

```diff
@@ -248,7 +248,10 @@ const isCompatibleDerivedProperty = (
 ): boolean =>
   o.name === name &&
   Array.isArray(o.parameters) &&
-  o.parameters.length === params.length;
+  // Note: In Engine this check is done by comparing the size of the derived property's `classifierGenericType` 's type arguments.
```

Comment:
Can we shorten the comment please? I think we do it for a lot of other places so ...